src/record/export/repositories/stringifiers/csvStringifier/index.ts

```diff
@@ -0,0 +1,64 @@
+import type { LocalRecord } from "../../../types/record";
+import type { CsvRow } from "../../../../../kintone/types";
+import type { RecordSchema } from "../../../types/schema";
+
+import stringify from "csv-stringify";
+import Pumpify from "pumpify";
+
+import { convertRecord, recordReader } from "./record";
+import { LINE_BREAK, SEPARATOR } from "./constants";
+import { buildHeaderFields } from "./header";
+import type { TransformCallback } from "stream";
+import { PassThrough, Transform } from "stream";
+import type { Stringifier } from "../index";
+
+export class CsvStringifier extends Pumpify.obj implements Stringifier {
```

Comment:
Extending third party objects is not safe as the exported interfaces are not stable and I think such classes are not designed for such usage.  Please consider [delegation pattern](https://en.wikipedia.org/wiki/Delegation_pattern).

Reply:
Or, I propose another solution.  I think we can replace pumpify with a `pipe()` chain.  This mechanism enables us to combine multiple streams to one.  

```typescript
this.stream = new PassThrough({ objectMode: true });

// pipe
this.stream
  .pipe(this.recordTransform)
  .pipe(this.csvStringifier)
  .pipe(this.encoder)
  .pipe(w);

// write
this.stream.write(records);
```

Please let me check the comparison between pumpify and pipe-chain later :)